tests/unit/workspace_access/test_tacl.py

```diff
@@ -92,15 +94,14 @@ def test_tacl_crawler_multiple_permissions():
             ]
         }
     )
-    tables_crawler = TablesCrawler(sql_backend, "test")
-    udf_crawler = UdfsCrawler(sql_backend, "test")
-    grants_crawler = GrantsCrawler(tables_crawler, udf_crawler)
+    grants_crawler = GrantsCrawler(TablesCrawler(sql_backend, "test"), UdfsCrawler(sql_backend, "test"))
```

Comment:
With the additional assertions further below (to satisfy the type-checker), this change was needed to get the statement count for the function back under 50 (pylint rule).